audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+        "        enable_automatic_punctuation=True,\n",
+        "    ),\n",
+        "    auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SKf3pMBl5E4f"
+      },
+      "source": [
+        "#### Set the audio file you want to transcribe\n",
+        "\n",
+        "For the batch transcription, you need the audio be staged in a Cloud Storage bucket. Then you set the associated metadata to pass in the batch recognition request."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "o1VCvEEI5MkG"
+      },
+      "outputs": [],
+      "source": [
+        "audio_metadata = cloud_speech.BatchRecognizeFileMetadata(\n",
+        "    uri=INPUT_LONG_AUDIO_SAMPLE_FILE_URI\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5HOKZLp25yFB"
+      },
+      "source": [
+        "#### Define batch recognition request\n",
+        "\n",
+        "Next, you define the batch recognition request. Notice how you define a recognition output configuration which allows you to determine how would you retrieve the resulting transcription outcome."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "SItkaX7tyZ14"
+      },
+      "outputs": [],
+      "source": [
+        "batch_recognition_request = cloud_speech.BatchRecognizeRequest(\n",
+        "    recognizer=f\"projects/{PROJECT_ID}/locations/{LOCATION}/recognizers/_\",\n",
+        "    config=batch_recognition_config,\n",
+        "    files=[audio_metadata],\n",
+        "    recognition_output_config=cloud_speech.RecognitionOutputConfig(\n",
+        "        inline_response_config=cloud_speech.InlineOutputConfig(),\n",
+        "    ),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "YQY1eqaY7H0n"
+      },
+      "source": [
+        "#### Run the batch recognition request\n",
+        "\n",
+        "Finally you submit the batch recognition request which is a long term operation as you see below."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "AlZwRlLo6F1p"
+      },
+      "outputs": [],
+      "source": [
+        "operation = client.batch_recognize(request=batch_recognition_request)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "DrqsNzVmeWu0"
+      },
+      "outputs": [],
+      "source": [
+        "while True:\n",
+        "    if not operation.done():\n",
+        "        print(\"Waiting for operation to complete...\")\n",
+        "        time.sleep(5)\n",
+        "    else:\n",
+        "        print(\"Operation completed.\")\n",
+        "        break"
```

Comment:
I don't think you need this extra polling.

This code (similar to the next cell) does all of the polling with exponential backoff/retry.
```py
operation.result(timeout=timeout)
```

Reply:
I know. I just want to hightlight that it is running a long term op